flink-libraries/flink-table/pom.xml

```diff
@@ -92,6 +92,11 @@ under the License.
 			</exclusions>
 		</dependency>
 
+		<dependency>
+			<groupId>org.apache.hive</groupId>
```

Comment:
Hi @twalthr .After moved the code to flink-hcatalog, I found the `mvn clean verify` will fail the test. The reason is that the code depends on the FLINK-5881. However, the flink-hcatalog depends on the `1.3-SNAPSHOT` of the maven repository. The patch of FLINK-5881 wasn't available there.